app/queries/tax_rates_query.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class TaxRatesQuery < BaseQuery
```

Comment:
Non blocking comment, 
It's not specified in specs but I'm wondering if we want to add the ability to search by rate value.
Like, I want all 20% tax rates to be returned so I type `20`, WDYT?
Again, just thought about it and don't want to add more work on your plate 😬